src/main/java/org/broadinstitute/consent/http/mail/message/NewLibraryCardIssuedMessage.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import java.util.Map;
+import org.broadinstitute.consent.http.enumeration.EmailType;
+import org.broadinstitute.consent.http.models.User;
+
+public class NewLibraryCardIssuedMessage extends MailMessage {
+  private static final String NEW_LIBRARY_CARD_ISSUED = "Library Card Issued in DUOS.";
```

Comment:
Other messages seem to use sentence case, and omit period.

```suggestion
  private static final String NEW_LIBRARY_CARD_ISSUED = "Library card issued in DUOS";
```